test/unit/language-extensions/__snapshots__/multi.spec.ts.snap

```diff
@@ -54,24 +58,78 @@ exports[`invalid $multi call (({ $multi });): code 1`] = `"local ____ = nil"`;
 
 exports[`invalid $multi call (({ $multi });): diagnostics 1`] = `"main.ts(2,12): error TSTL: The $multi function must be called in an expression that is returned."`;
 
-exports[`invalid $multi call (const [a = 0] = $multi()): code 1`] = `""`;
+exports[`invalid $multi call (const [a = 0] = $multi()): code 1`] = `
+"a = ____(_G)
+if a == nil then
+    a = 0
+end"
+`;
+
+exports[`invalid $multi call (const [a = 0] = $multi()): diagnostics 1`] = `"main.ts(2,25): error TSTL: The $multi function must be called in an expression that is returned."`;
+
+exports[`invalid $multi call (const {} = $multi();): code 1`] = `
+"local ____ = {
+    ____(_G)
+}"
+`;
+
+exports[`invalid $multi call (const {} = $multi();): diagnostics 1`] = `"main.ts(2,20): error TSTL: The $multi function must be called in an expression that is returned."`;
```

Comment:
This diagnostic seems unclear, would it be clearer to change it to `The $multi function must be called in a return statement or variable declaration.`?

Reply:
This is what it was previously. $multi can't be called in a variable declaration, so maybe we change it to just `The $multi function must be called in a return statement`.